### PR TITLE
Add missing "Usage" for `pl$` functions

### LINKED ISSUE #560: Documents of `pl$func()` have no `Usage` section
Perhaps we shouldn't make function definitions directly to `pl$func`.

### PR DESCRIPTION
**Pull request by etiennebacher**
Close #560

### CODE REVIEW

**Comment by eitsupi** [comment]
I was imagining a way to add the `pl_` prefix to all functions and register them to `pl` in `zzz.R` like everything else.
Is it not possible to do it like that?

**Comment by etiennebacher** [comment]
Good idea, I think it should work

File: R/after-wrappers.R [hunk 1]
```diff
@@ -208,8 +208,10 @@ class(pl) = c("pl_polars_env", "environment")
 #' @return NULL
 #' @keywords functions
 #' @examples
+#' \dontrun{
 #' pl$show_all_public_functions()
-pl$show_all_public_functions = function() {
+#' }
```
Comment (eitsupi) [inline R/after-wrappers.R:213]: Why need this?

File: tests/testthat/_snaps/after-wrappers.md [hunk 2]
```diff
@@ -36,20 +36,21 @@
...
-      [77] "scan_ipc"                  "scan_ndjson"              
-      [79] "scan_parquet"              "select"                   
-      [81] "set_global_rpool_cap"      "set_options"              
-      [83] "show_all_public_functions" "show_all_public_methods"  
-      [85] "std"                       "struct"                   
-      [87] "sum"                       "sum_horizontal"           
-      [89] "tail"                      "threadpool_size"          
-      [91] "using_string_cache"        "var"                      
-      [93] "when"                      "with_string_cache"        
+      [67] "pub_class_env"             "pub_env"
```
Comment (eitsupi) [inline tests/testthat/_snaps/after-wrappers.md:39]: I feel that `pub_class_env` and `pub_env` do not need to be made public.
Is it possible to remove the `pl_` prefix from these original names?

Comment (etiennebacher) [reply tests/testthat/_snaps/after-wrappers.md]: Done

File: R/after-wrappers.R (hunk 1 above)
Comment (etiennebacher) [reply R/after-wrappers.R]: Some stuff I tried but forgot to remove, thanks

**Review by eitsupi — DISMISSED** [review]
Thanks for working on this!

**Review by eitsupi — APPROVED** [review]

**Pull request merged**